### DISCUSSION TITLE
Mão na massa cronômetro

### DISCUSSION THREAD

**Discussion by InacioCarvalhoOliveira** [balta-io/2801#96] [Perguntas e Respostas] [answered]
Salve balta, estou acompanhando o curso do básico pra dar uma reforçada e fiz a lógica do tempo (para segundo e minuto) da seguinte forma, o que você achou?

segue: 

            int tempSegundo = 1;
            int tempMinuto  = tempSegundo * 60;
            
            switch (type)
            {
                case 's': Start(tempo * tempSegundo);                     
                    break;

                case 'm':Start(tempo * tempMinuto);
                    break;
                    
                default:
                    break;
            }

**Comment by andrebaltieri** [comment, marked as answer]
@InacioCarvalhoOliveira me pareceu bom!!! Obrigado por compartilhar 💜
